src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final int INDEX_OPERATION_TYPE = 0;
+    public static final int INDEX_AMOUNT = 1;
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        readData(fromFileName);
+        generateReport(toFileName);
```

Comment:
Better divide process for 3 steps and methods: readData, createReport, writeData